Cargo.toml

```diff
@@ -24,3 +24,9 @@ exclude = [
 [profile.release.package.tendermint-light-client-js]
 # Tell `rustc` to optimize for small code size.
 opt-level = "s"
+
+# WORKAROUND for fizyk20/generic-array#133
+# Docs build is broken without this until above PR is merged/released
+[patch.crates-io.generic-array]
+git = "https://github.com/fjarri/generic-array"
+branch = "zeroize-docs"
```

Comment:
I added this as a workaround which fixes the docs build in CI, but it's not a great fix

Reply:
The fix was [just released](https://github.com/fizyk20/generic-array/pull/133#issuecomment-1486018064) in `generic-array` 0.14.7, so I took the liberty of removing the patched crate.